JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -152,7 +152,7 @@ public function normalize($object, $format = null, array $context = [])
                 $subResource = $this->resourceResolver->getResourceFromType($collectionType)
             ) {
                 $values = [];
-                foreach ($attributeValue as $index => $obj) {
+                foreach ($attributeValue->getValues() as $index => $obj) {
```

Comment:
AFAIK there is no `Collection` interface in Symfony. Only in Doctrine collection. And this serializer can be used with other kind of `\Traversable` such as plain old arrays.